pkg/proto/v0/settings.pb.micro_test.go

```diff
@@ -1123,53 +1239,3 @@ func TestSaveGetListSettingsValues(t *testing.T) {
 		})
 	}
 }
-
-//same as test above but List returns an empty array in case the extension name contains a `../`
```

Comment:
I cannot see where that test-case was added to the previous test, we should still test it

Reply:
The List request doesn't return an empty list anymore when the provided `extension` param is invalid. It returns a validation error, and that is already covered in the bundle properties test.